src/components/Accordion/index.tsx

```diff
@@ -17,9 +17,10 @@ export interface AccordionProps {
   className?: string;
   /** Unique id passed to data-qa-id property of the outermost wrapper of the component. Used for automation testing  */
   dataQa?: string | null;
+  noPadding?: boolean;
 }
 
-export const Accordion = ({ title, className, children, dataQa }: AccordionProps) => {
+export const Accordion = ({ title, className, children, dataQa, noPadding = false }: AccordionProps) => {
```

Comment:
Is it still necessary to put `false` value even though the `noPadding` props is already optional props?

Reply:
just wanna make it clearer in the docs. the props table has a "Default" column that lists the default value of each props

Reply:
I see